fiftyone/utils/uid.py

```diff
@@ -0,0 +1,44 @@
+"""Utilities for usage analytics
+
+| Copyright 2017-2020, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+import os
+import uuid
+
+import fiftyone.constants as foc
+
+
+_TRACKING_NOTE = """
+FiftyOne tracks UUID based import usage by default. We are a small team looking
+to grow, and usage data is critical to that end.
+
+To disable tracking, use the `FIFTYONE_DO_NOT_TRACK=true` environment
+variable.
+"""
+
+
+def _get_user_id():
```

Comment:
Leaving myself a note to investigate when this message is printed during normal usage